sdk/containerregistry/azure-containers-containerregistry/src/test/resources/session-records/ContainerRegistryClientIntegrationTests.audienceTest[1].json

```diff
@@ -1,54 +1,54 @@
 {
   "networkCallRecords" : [ {
     "Method" : "POST",
-    "Uri" : "https://REDACTED.azurecr.io/oauth2/exchange?api-version=2021-07-01",
+    "Uri" : "https://REDACTED.azurecr.io/oauth2/exchange?api-version=V2021_07_01",
```

Comment:
Was the missing `V` in `api-version` a bug?

Reply:
Hmm...There is something funky going on in the API. Let me double check. This works on live test because the service does not actually take the Api-version at all just makes it look like that in the rest api.